### PR TITLE
Added Header

### PR DESCRIPTION
**Pull request by yilinliu-7**
Screenshot 2026-01-16 221543
On figma the white layer over the picture is at 85% opacity but it looks less transparent on react with the same percentage. Should I change it to match figma visually or keep it at 85%?

### CODE REVIEW

File: src/components/header.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  title: string;
+}
+
+function Split(title: string) {
+  const name = title.split(" ");
+  return {
+    first: name[0],
+    second: name.slice(1).join(" "),
+  };
+}
```
Comment (j1munoz) [inline src/components/header.tsx]: This is a good approach, but there is also another way of executing this in a more secure way. This splits the entire string into two parts: the first word and the rest of the string. However, what if we want to have the top part of the header be more than one word? It won't happen since it's only using the first word as the top part of the header. Here's another approach:
- In `HeaderProps`, add in the following field: `subTitle?: string;`. The `?` means that this field is optional, so the `subTitle` prop does not have to be included when you call in the component, and it won't give you any errors.
- Remove the `Split` function

**Comment by yilinliu-7** [comment]
Just another question, on figma it looks like there's another color mixed in at the middle of the gradient, but I've tested all the possible colors and none of them looks similar to the effect on figma. Is the current gradient pattern fine as it is right now?

**Review by j1munoz — APPROVED** [review]
Looks great!! And yes, the gradient pattern looks great too
